practices/graded-browser-support.md

```diff
@@ -11,9 +11,9 @@ We follow the guidelines for graded browser support outlined by Nate Koechley at
 
 Our browser support in summary:
 
-- Grade-A are browser versions that are explicitly *supported*. A product bug in one of these browsers is a high priority.
-- Grade-C are browser versions that are explicitly *_not_ supported*. We serve only the minimum styling and functionality to these browsers. 
-- Grade-X are the grey area between Grade-A and Grade-C. We serve them the same code as Grade-A, but we do not test our products against these browser versions. Over time, analysis of errors and usage patterns may cause some browsers in Grade-X to be deliberately demoted to Grade-C. 
+- Grade-A are modern and/or popular browser versions that receive all styling and behaviour. Bugs are addressed with high priority.
+- Grade-C are old and/or uncommon browser versions which are known to break if receiving full styling and behaviour. We serve only minimum CSS and JavaScript (detailed below). Bugs are addressed with high priority.
+- Grade-X is the grey area between Grade-A and Grade-C. We serve them the same code as Grade-A, but we do not test our products against these browser versions. We expect our sites to work in these browsers. Over time, analysis of errors and usage patterns may cause some browsers in Grade-X to be deliberately changed to either Grade-A or Grade-C, to allow for better support. 
```

Comment:
changing this to "Grade-X _**is**_..." means that the next sentence "we serve _**them**_" is no longer grammatical 😉

Reply:
ARGH. Thanks :)